integration_test/dataset/wanedge.yml

```diff
@@ -144,13 +160,18 @@ translations:
   - uuid: tr-saddr
     interface_uuid: if-glo2vnetgw
     mode: static_address
-    passthrough: true
+    passthrough: false
```

Comment:
We're not testing passthrough and the correct setting for SNAT should be with no passthrough. Only addresses that have translations should pass.